### PR TITLE
doc(readme): update readme

### PR DESCRIPTION
**Pull request by Helias**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,52 +65,41 @@ then configure it.
 npm run web-dev
 ```
 
-Wowser will be served on `http://localhost:8080`.
+Wowser will be served on `http://0.0.0.0:8080/webpack-dev-server/`.
```
Comment (Yehonal) [inline README.md]: 0.0.0.0 is not an accessible url for windows, I would say: is listening on 0.0.0.0 and you can access locally via 127.0.0.1

File: README.md [hunk 2]
```diff
@@ -71,52 +65,41 @@ then configure it.
...
+#### - Proxy port for a public server
 
+Run proxy for a public server:
 ```shell
-npm run proxy 3724 host:3724
-npm run proxy 8129 host:8129
+npm run proxy 3724 server.realmlist:3724
+npm run proxy 8085 server.realmlist:8085
 ```
+replacing *server.realmlist* with the realmlist of the server.
```
Comment (Yehonal) [inline README.md:101]: We could add "the trick" to test it locally here

**Review by Yehonal — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Helias) [reply README.md:101]: I already added it above

File: README.md (hunk 1 above)
Comment (Helias) [reply README.md]: ok

**Pull request merged**